templates/nginx_reverse_proxy.j2

```diff
@@ -1,39 +1,48 @@
+# {{ ansible_managed }}
+{# Add local_test_enviroment top level domain to domain string in local testing. #}
+{% macro add_local(domain) -%}
+    {{domain}}{{local_test_environment|default('')}}
+{%- endmacro %}
+
 server {
-    {% for definition in nginx_listen %}listen {{definition}};
-    {% endfor %}
+{% for definition in nginx_listen %}
```

Comment:
How does the formatting of the generated file look with this?

Reply:
Looks like this, I don't think it changed from that change.

```
server {
    listen 80;
    listen 443 ssl;

...
```